docarray/typing/tensor/tensorflow_tensor.py

```diff
@@ -0,0 +1,141 @@
+from typing import TYPE_CHECKING, Any, Dict, Generic, Type, TypeVar, Union
+
+import numpy as np
+import tensorflow as tf
+
+from docarray.typing.proto_register import _register_proto
+from docarray.typing.tensor.abstract_tensor import AbstractTensor
+
+if TYPE_CHECKING:
+    from pydantic.fields import ModelField
+    from pydantic import BaseConfig
+    from docarray.proto import NdArrayProto
+    from docarray.computation.tensorflow_backend import TensorFlowCompBackend
+
+from docarray.base_document.base_node import BaseNode
+
+T = TypeVar('T', bound='TensorFlowTensor')
+ShapeT = TypeVar('ShapeT')
+
+tf_base: type = type(tf.Tensor)
+node_base: type = type(BaseNode)
+
+
+class metaTensorFlow(
+    AbstractTensor.__parametrized_meta__,  # type: ignore
+    node_base,  # type: ignore
+    tf_base,  # type: ignore
+):  # type: ignore
+    pass
+
+
+@_register_proto(proto_type_name='tensorflow_tensor')
+class TensorFlowTensor(AbstractTensor, Generic[ShapeT], metaclass=metaTensorFlow):
+
+    __parametrized_meta__ = metaTensorFlow
+
+    def __init__(self, tensor: tf.Tensor):
+        super().__init__()
+        self._tensor = tensor
```

Comment:
Should we override `__getattr__()` to pass through all calls to this `self._tensor()`?

Reply:
what i mean is that right now our tensorflow tensor is not a subclass of the native tensorflow tensor. But we could fake the same feeling by passing through all operations to `.tensor`. Something like this:

```
def __getattr__(self, name):
    return getattr(self.tensor, name)
```

Reply:
ok then let's not do it, it's too dangerous